### PR TITLE
Add scripts to start backend services

### PR DESCRIPTION
**Pull request by sdkottegoda**
## Purpose
Adds scripts to start backend services

### CODE REVIEW

File: product-scenarios/backend-service/ballerinaService/resources/json/request/tmp.json [hunk 1]
```diff
@@ -0,0 +1 @@
...}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"
```
Comment (milindaperera) [inline product-scenarios/backend-service/ballerinaService/resources/json/request/tmp.json]: Let's remove this temporary json message

File: product-scenarios/backend-service/start_rest_services.sh [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        rm -r $TOMCAT_HOME/webapps/axis2*
+        cp $BACKEND_LOCATION/stockQuoteService/axis2.war $TOMCAT_HOME/webapps/
+        sh $TOMCAT_HOME/bin/startup.sh
+        echo "started stock quote service"
+
+    fi
+fi
+
+echo "############################################# Started Services ##################################################"
+
```
Comment (milindaperera) [inline product-scenarios/backend-service/start_rest_services.sh]: Let's schedule cron job at the end after setting up environment

File: product-scenarios/backend-service/ballerinaService/resources/json/request/tmp.json (hunk 1 above)
Comment (sdkottegoda) [reply product-scenarios/backend-service/ballerinaService/resources/json/request/tmp.json]: done

File: product-scenarios/backend-service/start_rest_services.sh (hunk 2 above)
Comment (sdkottegoda) [reply product-scenarios/backend-service/start_rest_services.sh]: done

**Pull request merged**